arch/arm64/boot/dts/freescale/crocodile.dtsi

```diff
@@ -431,8 +431,8 @@
 &usbotg1 {
 	dr_mode = "otg";
 	disable-over-current;
-	picophy,pre-emp-curr-control = <3>;
-	picophy,dc-vol-level-adjust = <7>;
+	samsung,picophy-pre-emp-curr-control = <3>;
+	samsung,picophy-dc-vol-level-adjust = <7>;
```

Comment:
Is it something that is already available upstream? If not - then we should not change those bindings until they are landing upstream.

Reply:
yes, they are available [upstream](https://github.com/torvalds/linux/commit/8977c947b321858ad878106e6c1dbfb719f23db6)

Reply:
Why is it required?